docs/src/concepts/smartweave.md

```diff
@@ -0,0 +1,26 @@
+# SmartWeave
+
+## What is SmartWeave?
+
+The SmartWeave protocol is the smartcontract protocol for Arweave. The major difference from other blockchains - namely Ethereum, one of the leaders in smartcontract ecosystems - is Arweave nodes do not validate smartcontracts, but rather the user themselves when they interact with the contract, thru the use of "lazy-evaluation". SmartWeave contracts currently support JavaScript, TypeScript, Rust, Go, AssemblyScript, and WASM (WebAssembly).
```

Comment:
Maybe something like...
> SmartWeave is them name given to the dominant SmartContract paradigm on Arweave. A unique property of SmartWeave contracts is that the current state of the contract is provided by a process of "lazy evaluation".  This means that instead of Arweave mining nodes constantly evaluating the current state of all contracts, a client reading a contract evaluates the state for themselves.  

repeating "protocol" 2x in the first sentence throws me off.